lang/sql/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Style Guide for SQL
+pageid: lang-sql
+---
+
+
+Keywords
+--------
+
+All keywords should be uppercase.
+
+    SELECT COUNT(*)
+    WHERE DATEDIFF(NOW(), someTable.timeStamp)
+    ORDER BY ASC
+
+
+Indentation and Whitespace
+--------------------------
+
+Clauses should be separated by newlines.
+
+    SELECT *
+    FROM someTable
+    ORDER BY someField DESC
+
+When using "SELECT" on multiple columns, place the columns on the same line.
```

Comment:
I think backticks are more appropriate for quotes here and for the next paragraph.  You're citing the code.